benchmarks/distributed/test_distributed.py

```diff
@@ -202,3 +204,17 @@ def f3():
 print(f"Task time: {task_time} ({MAX_RUNNING_TASKS_IN_CLUSTER} tasks)")
 print(f"PG time: {pg_time} ({MAX_PLACEMENT_GROUPS} placement groups)")
 print(f"Node launch time: {launch_time} ({MAX_NUM_NODES} nodes)")
+
+if "TEST_OUTPUT_JSON" in os.environ:
+    out_file = open(os.environ["TEST_OUTPUT_JSON"], "w")
+    results = {
+        "actor_time": actor_time,
+        "num_actors": MAX_ACTORS_IN_CLUSTER,
+        "task_time": task_time,
+        "num_tasks": MAX_RUNNING_TASKS_IN_CLUSTER,
+        "pg_time": pg_time,
+        "num_pgs": MAX_PLACEMENT_GROUPS,
+        "launch_time": launch_time,
+        "num_nodes": MAX_NUM_NODES
```

Comment:
Can you add success: [1|0] to the result? I am thinking to use them to just see if tests succeed very quickly

Reply:
Sure, it will be hardcoded though because the failure condition for all these tests is a timeout/cluster crash/exception thrown.

Reply:
I see. Is it impossible to isolate the application level bugs here?

Reply:
what would you consider an "application level bug" for these tests?

Reply:
Maybe failures that happen due to assertion failure or exceptions. When tests fail, I just would like to know if that's due to releaser failure or test failure (for timeout, it is not possible to differentiate them, but for assertion error sort of things it should be possible).

Reply:
I see, there are assertions in the tests, but the test won't have any output if the assertion occurs (the rest of the tests won't run because they're really just asserting the system is in a consistent state).